index.js

```diff
@@ -5,17 +5,10 @@ var fs   = require('fs');
 
 exports.name = 'jade';
 exports.outputFormat = 'html';
-exports.compile = function (source, options) {
-  var fn = jade.compile(source, options);
-  return {fn: fn, dependencies: fn.dependencies}
-};
-exports.compileClient = function (source, options) {
-  return jade.compileClientWithDependenciesTracked(source, options);
-};
-exports.compileFile = function (path, options) {
-  var fn = jade.compileFile(path, options);
-  return {fn: fn, dependencies: fn.dependencies}
-};
+
+exports.compile = jade.compile;
+exports.compileClient = jade.compileClientWithDependenciesTracked;
+exports.compileFile = jade.compileFile;
```

Comment:
No. Jstransformer handles the case of just returning a function when dependencies are not known (or can't be tracked for some reason).  The format of returning a function that has a dependencies property is Jade specific.